build.sbt

```diff
@@ -17,7 +17,7 @@ lazy val codacyEngineScalaSeed = project
     ),
     name := "codacy-engine-scala-seed",
     // App Dependencies
-    libraryDependencies ++= Seq(Dependencies.playJson, Dependencies.codacyPluginsApi, Dependencies.betterFiles),
+    libraryDependencies ++= Seq(Dependencies.playJson, Dependencies.playJsonDerivedCodecs, Dependencies.codacyPluginsApi, Dependencies.betterFiles),
```

Comment:
Put play json and derived codec together or remove the playJson dependency as is already comes from playJsonDerivedCodecs.